api/v1/testkube.yaml

```diff
@@ -3289,6 +3289,545 @@ paths:
                 items:
                   $ref: "#/components/schemas/Problem"
 
+  /test-workflows:
```

Comment:
I would add comments, that theses are Pro methods

Reply:
Good point, we could add `pro` tag to them 👍

Reply:
Added tag in https://github.com/kubeshop/testkube/pull/5058 👍